### PR TITLE
[1LP][RFR]tests for system schedules 

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
{{pytest: cfme/tests/configure/test_schedule_operations.py}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 33fa1c5b738a03b66606d9f2582da285949aadd0:

`cfme/tests/configure/test_schedule_operations.py`:
- :bangbang: Line 8:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 12:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 13:41: **E226** *missing whitespace around arithmetic operator*
- :bangbang: Line 31:9: **E123** *closing bracket does not match indentation of opening bracket's line*
- :heavy_exclamation_mark: Line 91:1: **W391** *blank line at end of file*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 092050418a5882eee4273071e84863247a326f02:

`cfme/tests/configure/test_schedule_operations.py`:
- :bangbang: Line 56:9: **E123** *closing bracket does not match indentation of opening bracket's line*
- :bangbang: Line 119:4: **E121** *continuation line under-indented for hanging indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6ec7941d60199bb8556b3ada5cb8bc9c0c544ead:

`cfme/tests/configure/test_schedule_operations.py`:
- :bangbang: Line 64:9: **E123** *closing bracket does not match indentation of opening bracket's line*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 762398d2c10eedf48748b4d2982879c690573476:

`cfme/tests/configure/test_schedule_operations.py`:
- :red_circle: Line 4:1: **F401** *'datetime.datetime' imported but unused*
- :bangbang: Line 155:73: **E226** *missing whitespace around arithmetic operator*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/configure/test_schedule_operations.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    start_date = current_time + relativedelta.relativedelta(days=2)
+    schedule = appliance.collections.system_schedules.create(
+        name=fauxfactory.gen_alphanumeric(),
+        description=fauxfactory.gen_alphanumeric(),
+        start_date=start_date
+    )
+
+    view = appliance.browser.create_view(BaseLoggedInPage)
+    view.flash.assert_message('Schedule "{}" was saved'.format(schedule.name))
+    # test for bz
```
Comment (otsuman) [inline cfme/tests/configure/test_schedule_operations.py]: **REQUIRED**: Can you provider BZ number, please?

File: cfme/tests/configure/test_schedule_operations.py [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+@pytest.mark.parametrize('run_types', run_types, ids=[type[0] for type in run_types])
+def test_schedule_timer(appliance, run_types, host_with_credentials, request, current_server_time):
+
+    run_time, time_diff, time_num = run_types
+    current_time, tz_num = current_server_time
+    start_date = current_time + relativedelta.relativedelta(minutes=5)
+    view = navigate_to(appliance.collections.system_schedules, 'Add')
+    available_list = view.time_zone.all_options
+    # bz is here
```
Comment (otsuman) [inline cfme/tests/configure/test_schedule_operations.py]: Same here

File: cfme/configure/configuration/system_schedules.py [hunk 3]
```diff
@@ -197,20 +198,42 @@ def update(self, updates, reset=False, cancel=False):
...
-            flash_message = 'All changes have been reset'
         if cancel:
             view.cancel_button.click()
-            flash_message = 'Edit of Schedule "{}" was cancelled by the user'.format(self.name)
-        elif updated:
+        elif updated and not cancel and not reset:
             view.save_button.click()
             view = self.create_view(ScheduleDetailsView, override=updates)
-            name = updates.get('name') if updates.get('name') else self.name
-            flash_message = 'Schedule "{}" was saved'.format(name)
```
Comment (mshriver) [inline cfme/configure/configuration/system_schedules.py]: :+1:

**Pull request merged**